lib/AST/ASTImporter.cpp

```diff
@@ -8033,9 +8042,9 @@ Expected<Decl *> ASTImporter::Import(Decl *FromD) {
       // traverse of the 'to' context).
       auto PosF = ImportedFromDecls.find(ToD);
       if (PosF != ImportedFromDecls.end()) {
-        if (LookupTable)
+        if (SharedState->getLookupTable())
           if (auto *ToND = dyn_cast<NamedDecl>(ToD))
-            LookupTable->remove(ToND);
+            SharedState->getLookupTable()->remove(ToND);
         ImportedFromDecls.erase(PosF);
       }
 
```

Comment:
For the next FIXME comment: We can add the text that the ImportDeclErrors in the shared state should mark all failed objects.

Reply:
Ok, that's a good idea.